internal/app/middleware/middleware_test.go

```diff
@@ -0,0 +1,98 @@
+package middleware_test
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/kripsy/shortener/internal/app/middleware"
```

Comment:
```suggestion
	"github.com/kripsy/shortener/internal/app/middleware"
	
```

Reply:
what change?)
it's separate type of import packeges)